API.md

```diff
@@ -44,6 +44,8 @@ A geocoder component using Mapbox Geocoding API
     -   `options.localGeocoder` **[Function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)?** A function accepting the query string which performs local geocoding to supplement results from the Mapbox Geocoding API. Expected to return an Array of GeoJSON Features in the [Carmen GeoJSON](https://github.com/mapbox/carmen/blob/master/carmen-geojson.md) format.
     -   `options.reverseMode` **(`"distance"` \| `"score"`)** Set the factors that are used to sort nearby results. (optional, default `'distance'`)
     -   `options.reverseGeocode` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)?** Enable reverse geocoding. Defaults to false. Expects coordinates to be lat, lon.
+    -   `options.addToMap` **[Boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** Indicates that the location of the user-selected result should be added to the map as a [mapbox-gl#Marker](https://docs.mapbox.com/mapbox-gl-js/api/#marker) (optional, default `true`)
```

Comment:
Do you think naming this something like `showMarker` would be clearer by indicating this specifically relates to a marker? `addToMap` at first glance to me means add the geocolate control to the map, which is done automatically already.

Do you think it's clearer like this? That's the general style used commonly in GL JS docs, for example https://docs.mapbox.com/mapbox-gl-js/api/#popup

    "If `true`, a [Marker](...) will be added to the map at the location of the user-selected result."